### ISSUE TITLE
Rename buttons do not work

### ISSUE THREAD

**Issue by Hatbringer** [OlauncherCF/OlauncherCF#297]
### Checklist

- [X] I have searched for existing issues about this bug.
- [X] I know that if existing issues are available, this issue will be closed without comment, unless an explaination is given on why this new issue is needed.
- [X] I know that not filling out all information will greatly reduce the devolopers chances of debugging your issue

### Bug description

When attempting to rename an item on the home screen, pressing the rename button on the top right does not do anything.

### Steps to reproduce the bug

1. Home Screen
2. Long press pinned app name
3. Select "rename"

### OlauncherCF Version

v1.6.5

### Android Version

Android 13

### Phone

Google Pixel 4A

### Additional information

The rename button changes color in response to being pressed, butno action follows.

**Comment by Hatbringer** [comment]
> From the README:
> 
> > Renaming apps on the home screen is already supported. Just long-click on an app on the home screen and start typing
> 
> To rename the app, you have to enter a name, and only after that press the rename button. This is not the most intuitive way, I give you that. However, this is still the original way it was in OlauncherAF. And I have currently not the time to change this

Thanks! I would like to mention that he keybord only pops up by default with Behavior > Auto show keybord > on. With the option disabled, the keybord does not appear untill pressing the search entry at the start of the list.
